packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebsiteDataStoreHostApi.m

```diff
@@ -60,7 +60,7 @@ - (void)createDefaultDataStoreWithIdentifier:(nonnull NSNumber *)identifier
                       modifiedSince:[NSDate dateWithTimeIntervalSince1970:
                                                 modificationTimeInSecondsSinceEpoch.doubleValue]
                   completionHandler:^{
-                    completion(@(records.count > 0), nil);
+                    completion([NSNumber numberWithBool:(records.count > 0)], nil);
```

Comment:
Wow, that's a surprising foot-gun. TIL.

What type was it turning into exactly? Int?

Reply:
Yea, it was returning an int for some reason.